config/db.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "password",
+    "database": "chapter",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "port": 54320
+  },
+  "test": {
+    "username": "postgres",
+    "password": "password",
+    "database": "chapter",
```

Comment:
We should use a different db for testing, one that we bring up and tear down when testing. Not essential for this pr, just leaving a note